.github/workflows/maven.yml

```
@@ -22,3 +23,16 @@ jobs:
         java-version: 11
     - name: Build with Maven
       run: mvn -B package --file pom.xml
+
+  publish-app:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Push to GitHub Packages
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          registry: docker.pkg.github.com
+          repository: uecit/cactus-fhir-server/fhir-server
```

Could this be cactus-fhir-server in the image tag too, to indicate it's been built from this repo? The repeated naming grates on me, but then at least we're clear about which images are part of the project.

```suggestion
          repository: uecit/cactus-fhir-server/cactus-fhir-server
```